siriuspy/siriuspy/devices/injsys.py

```diff
@@ -9,7 +9,7 @@
 from .lillrf import DevLILLRF
 from .modltr import LIModltr
 from .pwrsupply import PowerSupply, PowerSupplyPU
-from .timing import EVG, Event, Trigger
+from .timing import Trigger, HLTiming
```

Comment:
it seems the Trigger device is not used anymore

Reply:
right, I removed it. Thanks!